gammapy/scripts/config/schema.yaml

```diff
@@ -342,7 +390,7 @@ if:
                     const: MapDataset
 then:
     properties:
-        geometry:
+        geom:
```

Comment:
Since `geom` is now nested into `reduction` this should also be reflected in the conditionals.

```
if:
    properties:
        reduction:
            properties:
                dataset-type:
                     const: MapDataset
then:
    properties:
        reduction:
            properties:
            required: [geom]
                geom:
                    required: [binsz, width, skydir, coordsys, proj, axes]
                    properties:
                        axes:
                        required: [e_reco]
```

Note that you may add also a conditional validation for `geom-irf` if you need it.

Reply:
Done